frontend/src/components/Comment/CommentList.js

```diff
@@ -1,28 +1,36 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useLayoutEffect } from 'react'
 import { getCommentByPicture } from './CommentAPI'
 import { Comment } from './Comment'
 import './SCSS/CommentList.scss'
+import Axios from 'axios'
 
```

Comment:
Was this import necessary? I might have missed something but I don't see where it is used.